### ISSUE TITLE
Gradle plugin error: Cannot add extension with name 'jbbp', as there is an extension already registered with that name

### ISSUE THREAD

**Issue by int02h** [raydac/java-binary-block-parser#41]
Plugin version: 2.0.4
Gradle: 7.4.2

I am trying to apply the JBBP plugin to a simple empty command line project and get this error.

Stacktrace:
```
Caused by: java.lang.IllegalArgumentException: Cannot add extension with name 'jbbp', as there is an extension already registered with that name.
	at org.gradle.internal.extensibility.ExtensionsStorage.add(ExtensionsStorage.java:40)
	at org.gradle.internal.extensibility.DefaultConvention.add(DefaultConvention.java:120)
	at org.gradle.internal.extensibility.DefaultConvention.create(DefaultConvention.java:138)
	at org.gradle.internal.extensibility.DefaultConvention.create(DefaultConvention.java:132)
	at com.igormaznitsa.jbbp.plugin.gradle.JBBPPlugin.apply(JBBPPlugin.java:14)
	at com.igormaznitsa.jbbp.plugin.gradle.JBBPPlugin.apply(JBBPPlugin.java:7)
```

I quickly checked out the source code [here](https://github.com/raydac/java-binary-block-parser/blob/722ff52f980a022ddc10201dd619a655fe02cbcb/jbbp-plugins/jbbp-gradle/src/main/java/com/igormaznitsa/jbbp/plugin/gradle/JBBPPlugin.java#L14) and I indeed see double creation of extension.

**Comment by raydac** [comment]
I have removed duplication, is it possible to check snapshot version on your side?

**Comment by int02h** [comment]
Yep, sure. Which version should I check?

**Comment by raydac** [comment]
I will add autotests for gradle to be sure that it works

**Comment by raydac** [comment]
I have published new version of Gradle plugin, 2.0.5

**Comment by int02h** [comment]
Yes, I confirm that 2.0.5 solved the issue. Thank you.
